Tests/BootCamp.Chapter.Tests/PokerTests.cs

```diff
@@ -0,0 +1,41 @@
+﻿using BootCamp.Chapter.Gambling;
+using BootCamp.Chapter.Gambling.Poker;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Xunit;
+
+namespace BootCamp.Chapter.Tests
+{
+    public class PokerTests
+    {
+        [Fact]
+        public void GetWinners_Given_List_Players_Returns_Winnner()
+        {
+            var player1 = new Player("player1", new PokerHand());
```

Comment:
PLease remember the essense of test- it needs to be self documenting. When I read 50 lines of code just to get to the setup, I will not be happy as a reader. Fix it, but extracting test steps into creation/assertion functions in case they grow like this!

Reply:
In this case, nonoe of this is needed. It should be just 3 lines of code. Also, this is not a minimalistic test, as there are more than enough things to check what you're checking...

Reply:
oke, 
Then I need to think very hard how I can do this.